app/templates/partials/g12_recovery_information.html

```diff
@@ -0,0 +1,11 @@
+{% if supplier.g12_recovery %}
+  <div class="wrapper">
+    {%
+      with
+      message = 'You have until 23:59 on 1 January 2525 to complete <a href="'|safe + url_for('.list_services', framework_slug='g-cloud-12') + '">your additional G12 services</a>.'|safe,
+      type = 'information'
+    %}
+      {% include 'toolkit/notification-banner.html' %}
```

Comment:
Could we use the new notification banner from the design system: https://design-system.service.gov.uk/components/notification-banner/ rather than the legacy toolkit?  However, if this is a lot of work may make more sense as a TODO?

Reply:
Hmm. I'm not very familiar with the design system. Do we have any example of how to use the design system banner?

Reply:
On further thought, I didn't think we'd ported supplier-frontend over to the Design System, so is this even possible?

Given that this should only exist in the codebase for a couple of months, I'm not sure we need to do any more than the bare minimum.

Reply:
@bjgill 
>  I didn't think we'd ported supplier-frontend over to the Design System, so is this even possible?

Yes, we do use `govuk-frontend` components alongside the old toolkit in supplier-frontend. Here is an example usage:  https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/master/app/templates/frameworks/legal_authority.html that said, this is a new component so I guess we may have to update dependencies to pick it up?

Reply:
I see what you mean. There exists a `banner` component that sounds as if it might be what we want: https://github.com/alphagov/digitalmarketplace-govuk-frontend/tree/master/src/digitalmarketplace/components/banner. However, I can see no examples of us previously using the `banner` component.

Would it be OK to merge this as-is, and make a note in the story to look into switching to the `govuk-frontend` component later?

Reply:
Actually - ^ is a DM component, not a govuk one.

There is a govuk component that does what we want, the `notification-banner`. However, that would need us to upgrade govuk-frontend to [3.10.0](https://github.com/alphagov/govuk-frontend/blob/master/CHANGELOG.md#3100-feature-release) - we're currently using 2.8.0.

Given the time pressure and temporary nature of this code, I don't think that this story should cover upgrading govuk-frontend. So, unless you object, I propose to leave as-is.

Reply:
@bjgill 👍  fine, perhaps we can add a TODO, in case we get around to switching this?

Reply:
Added.